test/all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. $DIR/common.sh
+
+for file in $(find "$DIR" -name "*.sh") ; do
+    if ! ( [[ "$file" == */all.sh ]] || [[ "$file" == */common.sh ]] ) ; then
```

Comment:
```bash
     if [[ "$file" != */all.sh && "$file" != */common.sh ]] ; then 
```

But really, just put the actual test files in their own directory, or give them a distinguished name (like `test-*.sh`).

https://github.com/CyberShadow/aconfmgr/tree/master/test/t